src/main/drivers/vtx_common.c

```diff
@@ -25,14 +25,40 @@
 #include "platform.h"
 #include "build/debug.h"
 
+
 #if defined(VTX_COMMON)
 
 #include "vtx_common.h"
+#include "fc/config.h"
+
+PG_REGISTER_WITH_RESET_TEMPLATE(vtxSettingsConfig_t, vtxSettingsConfig, PG_VTX_SETTINGS_CONFIG, 0);
```

Comment:
As a general principle I don't like to see the parameter group REGISTER and RESET function in the device driver, I'd rather see them in the IO layer. By all means define `vtxSettingsConfig_t` in `drivers/vtx_common.h`, but I'd move all the `PG_` items into the IO layer.

Reply:
So are you suggesting the recreation of vtx_settings_config.c/h for the purpose of declaring the PG?

Reply:
I was suggesting moving them to `io/vtx_common.c/h`

Reply:
10-4 will do :)